### ISSUE TITLE
eventlog size Security Log

### ISSUE THREAD

**Issue by PehDeh** [MicrosoftDocs/microsoft-365-docs#13024]
Hi,
can you please state what the Eventlog Size of the Security Log is as it does not work with Servers having a 4GB Security Log. Works with smaller Logs.
Support is not willing/able to give these information. But It will Timeout and not work if we have a 4 GB Security Log.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6efb7742-1171-00f4-ab40-6bcc479078ca
* Version Independent ID: 9a79f573-5f20-0e4c-822e-8af4ebf65422
* Content: [Take response actions on a device in Microsoft Defender for Endpoint](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/respond-machine-alerts?view=o365-worldwide#collect-investigation-package-from-devices)
* Content Source: [microsoft-365/security/defender-endpoint/respond-machine-alerts.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/respond-machine-alerts.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @diannegali
* Microsoft Alias: **diannegali**

**Comment by yogkumgit** [comment]
@diannegali Please help us in resolving this issue. Thanks

**Comment by PehDeh** [comment]
yes i refer to the Security.evtx which is is beeing collected and to large Max Size of an EVTX is 4 GB. The link you added is wrong you send a Link to Firewall log.

Specifies the maximum size of the event logs in bytes. Enter a value between 64 kilobytes (KB) and 4 gigabytes (GB). The value must be divisible by 64 KB (65536).

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/limit-eventlog?view=powershell-5.1

Also via UI
https://helpcenter.netwrix.com/bundle/Auditor_10.0/page/Content/Configure_IT_Infrastructure/Windows_Server/WS_Event_Log_Settings.htm

**Comment by officedocspr5** [comment]
To make it easier for you to submit feedback on articles on learn.microsoft.com, we're transitioning our feedback system from GitHub Issues to a new experience. 

As part of the transition, this GitHub Issue will be moved to a private repository. We're moving Issues to another repository so we can continue working on Issues that were open at the time of the transition. When this Issue is moved, you'll no longer be able to access it.

If you want to provide additional information before this Issue is moved, please update this Issue before December 15th, 2023.

With the new experience, you no longer need to sign in to GitHub to enter and submit your feedback. Instead, you can choose directly on each article's page whether the article was helpful. Then you can then choose one or more reasons for your feedback and optionally provide additional context before you select **Submit**.

Here's what the new experience looks like.

**Note**: The new experience is being rolled out across learn.microsoft.com in phases. If you don't see the new experience on an article, please check back later.

First, select whether the article was helpful:

![Image showing a dialog asking if the article was helpful with yes and no answers.](https://learn.microsoft.com/Office/media/internal/new-feedback-yes-no.jpg)

Then, choose at least one reason for your feedback and optionally provide additional details about your feedback:

| Article was helpful | Article was unhelpful |
|:--|:---|
|![Image showing a dialog asking how the article was helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-helpful.jpg) | ![Image showing a dialog asking how the article wasn't helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-unhelpful.jpg)|

Finally, select **Submit** and you're done!

**Comment by diannegali** [comment]
@PehDeh the size limitation for the entire compressed investigation package is 210MB.
